lib/SILOptimizer/Mandatory/DefiniteInitialization.cpp

```diff
@@ -783,6 +786,155 @@ void LifetimeChecker::diagnoseInitError(const DIMemoryUse &Use,
     diagnose(Module, TheMemory.getLoc(), diag::variable_defined_here, isLet);
 }
 
+static void injectHopToExecutorBefore(SILLocation loc,
+                                      SILBasicBlock::iterator &insertPt,
+                                      SILValue actor, bool needsBorrow) {
+  SILBuilder B(insertPt);
```

Comment:
This needs to be a SILBuilderWithScope. We no longer use SILBuilder. We just have never finished the transition. Also please use b or builder.